src/lib/getImportDeclaration.ts

```diff
@@ -1,13 +1,10 @@
-import { AST_NODE_TYPES } from "@typescript-eslint/utils";
-import { ParserServices } from "@typescript-eslint/utils";
-import { ImportDeclaration } from "@typescript-eslint/types/dist/ast-spec";
+import { AST_NODE_TYPES, TSESTree, ParserServices } from "@typescript-eslint/utils";
 import { Symbol } from "typescript";
-import { JSXOpeningElement } from "@typescript-eslint/utils/node_modules/@typescript-eslint/types/dist/ast-spec";
```

Comment:
After upgrading the dependencies, the import path was broken. So I changed it to import the types from `@typescript-eslint/utils`.